### ISSUE TITLE
I drew 10 pets / 100p, but the cards didnt display any pets when flipped and my farm doesnt show any new pets 

### ISSUE THREAD

**Issue by git-good-w** [git-goods/gitanimals#207]
If it helps, this is in my logs 
Warning: Missing `Description` or `aria-describedby={undefined}` for {DialogContent}.

Absolutely *Loving* this so far otherwise; thank you! <3

**Comment by git-good-w** [comment]
@BirdMachine
Thanks for reporting the issue!

**Comment by BirdMachine** [comment]
Sure thing! Looks like it was a timing issue though.... pets have finally appeared :) Sorry for my impatience (though, it might help to have a little thing pop up saying "Pets should Appear in your farm in the next few minutes" or such could be handy?)

**Comment by devxb** [comment]
Farm type may take up to 1 hour till appear pets.

Thank you for opinion. 👍
